src/Controls/samples/Controls.Sample.Droid/MainApplication.cs

```diff
@@ -6,7 +6,7 @@
 namespace Maui.Controls.Sample.Droid
 {
 	[Application]
-	public class MainApplication : MauiApplication<MyApp>
+	public class MainApplication : MauiApplication<Startup>
```

Comment:
This passes the startup to the whole system, and this class and related on iOS could totally be generated if need be if we decide to use source generators.